graphs/breathfirstsearch/breathfirstsearch_test.go

```diff
@@ -0,0 +1,78 @@
+package breathfirstsearch
+
+import (
+	"testing"
+)
+
+func Test_BreadthFirstSearch(t *testing.T) {
```

Comment:
Remove the underscore

```suggestion
func TestBreadthFirstSearch(t *testing.T) {
```